toolkit/resources/dracutmodules/90mountesppartition/mountesppartition-generator.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+set -x
+set -e
+
+echo "Running mountesppartition-generator.sh" > /dev/kmsg
+
+# type getarg > /dev/null 2>&1 || . /lib/dracut-lib.sh
+
+function updateVeritySetupUnit () {
+    systemdDropInDir=/etc/systemd/system
+    verityDropInDir=$systemdDropInDir/systemd-veritysetup@root.service.d
```

Comment:
We shouldn't hardcode this to root verity. Use `veritysetup-pre.target` instead of `systemd-veritysetup@root.service`.